### PR TITLE
Display score when game finishes.

### PR DESCRIPTION
**Pull request by khoin**
Here's a small edit :) Unused score variable.

### CODE REVIEW

**Comment by joa** [comment]
Nice! Let's see what @addicted2defrac has to say about it.

I found some inconsistencies in the code, please have a look at the comments and perform the necessary changes.

File: src/java/defrac/game/Game.java [hunk 1]
```diff
@@ -69,7 +69,7 @@ public void onEvent(SwipeEvent e) {
 
         Events.onKeyDown.add(new EventListener<KeyboardEvent>() {
             @Override
-            public void onEvent(KeyboardEvent e) {
+            public void onEvent(KeyboardEvent e) { System.out.println(score);
```
Comment (joa) [inline src/java/defrac/game/Game.java]: This seems like debug code. Please remove it.

File: src/java/defrac/game/Game.java [hunk 2]
```diff
@@ -136,7 +136,7 @@ private void gameWon() {
 
         state = GAME_WON;
 
-        finish("You won!");
+        finish("You won!", "Score: "+ Integer.toString(score));
```
Comment (joa) [inline src/java/defrac/game/Game.java]: Integer.toString is unnecessary here. Please use just "Score: "+score

File: src/java/defrac/game/Game.java [hunk 3]
```diff
@@ -144,15 +144,16 @@ private void gameOver() {
 
         state = GAME_OVER;
 
-        finish("Game over");
```
Comment (joa) [inline src/java/defrac/game/Game.java:147]: Same here. Unnecessary Integer.toString can be removed.

Comment (defrac) [inline src/java/defrac/game/Game.java:147]: Actually I don't think that the score is interesting at all. The games purpose is to get 2048.

**Comment by khoin** [comment]
@joa I've fixed the issues you mentioned.

**Pull request merged**